src/rpc/net_type.h

```diff
@@ -22,7 +22,7 @@ struct RPCBase {
   virtual void Finalize() = 0;
 
   /**
-   * @brief Communicator type: 'socket', 'tensorpipe', etc
+   * @brief Communicator type such as 'socket'.
```

Comment:
We only support socket, right? Can we remove it?

Reply:
yes but I choose to keep it as I'd like to keep this API for potential extention. From functional perspective, the whole class(not only this API) could be removed as only `socket` is supported for now. If we decide to remove such logic thoroughly, I prefer to do it in a separate PR.

Reply:
Yes, let's remove them all since we only have one option. If we need future extension we may redesign it.